src/Norm/CMakeLists.txt

```diff
@@ -7,31 +7,19 @@ add_subdirectory(genere)
 add_subdirectory(basetest)
 add_subdirectory(generetest)
 
-# build norm.jar
+# Add norm.jar target
```

Comment:
Je ne suis pas d'accord... l'idée ici est bien de construire un jar. ça definit effectivement une nouvelle cible, mais c'est pas vraiment ce qui nous intéresse. En l'occurence la cible n'est pas `norm.jar` mais `norm_jar`

Reply:
Every target has an associated "built object". CMake is oriented to *targets* and what we do in the `CMakeLists.txt` files is defining and configuring targets.

I can expand the comment : "Add norm_jar target (which builds norm.jar)".

Reply:
non je ne suis pas d'accord. Le commentaire initial est plus clair pour tout le monde

Reply:
How about: `Add norm.jar build target`

Reply:
I changed it to `Add norm.jar (CMake names the target norm_jar)`